includes/CAPx/Drupal/Processors/EntityProcessor.php

```diff
@@ -31,8 +31,176 @@ class EntityProcessor extends ProcessorAbstract {
    *   The new or updated wrapped entity.
    */
   public function execute($force = FALSE) {
+
+    // Set this force.
+    $this->force == $force;
+
+    try {
+      $multi = $this->getMapper()->getConfigSetting('multiple');
+    }
+    catch (\Exception $e) {
+      $multi = FALSE;
+    }
+
+    // Sometimes we need to create one, other times we need moar.
+    if (!empty($multi) && $multi == 1) {
+      $entity = $this->executeMultiple();
+    }
+    else {
```

Comment:
What happens if `$multi` is empty or `0` or a negative integer here? Does that get caught as an exception, above?